tensorflow/c/experimental/stream_executor/stream_executor_internal.h

```diff
@@ -37,6 +38,16 @@ port::Status InitStreamExecutorPlugin(void* dso_handle);
 // testing).
 port::Status InitStreamExecutorPlugin(SEInitPluginFn init_fn);
 
+namespace {
+struct TFStatusDeleter {
+  void operator()(TF_Status* s) const { TF_DeleteStatus(s); }
+};
+using OwnedTFStatus = std::unique_ptr<TF_Status, TFStatusDeleter>;
+}  // namespace
```

Comment:
No anonymous namespaces in headers please: https://google.github.io/styleguide/cppguide.html#Unnamed_Namespaces_and_Static_Variables

Reply:
done.

Reply:
@sanjoy I have changed the code as your request, could you help to take a look? If there is any thing need to change, please let me know. Thanks.